BorrowBookControl.java

```diff
@@ -41,16 +41,16 @@ public void swiped(int memberId) {
             return;
         }
         if (this.library.MEMBER_CAN_BORROW(this.member)) {
-            this.pending = new ArrayList<>();
+            this.borrowBookStatus = new ArrayList<>();
             this.UI.setState(BorrowBookUI.UIState.SCANNING);
             this.state = ControlState.SCANNING; }

```

Comment:
D14: (lines 46 and 49) move each closing bracket '}' to a new line (each should have their own line) to meet code style guidelines requirements.
Check over both files to ensure this rule is consistently enforced throughout.

Reply:
fixed